cpg_workflows/large_cohort/combiner.py

```diff
@@ -70,6 +69,14 @@ def run(out_vds_path: Path, tmp_prefix: Path, *sequencing_group_ids) -> hl.vds.V
         sequencing_groups = [s for s in sequencing_groups if s in sequencing_group_ids]
 
     sequencing_groups = _check_gvcfs(sequencing_groups)
+    seq_types = set(s.sequencing_type for s in sequencing_groups)
+    if len(seq_types) > 1:
+        sequencing_type = config_retrieve(['workflow', 'sequencing_type'])
+        logging.warning(
+            f'Found multiple sequencing types for large cohort combiner: {seq_types}, using the config-provided {sequencing_type}',
+        )
```

Comment:
this one is spicy, for now I reckon this should be a log/fail with a ValueError. There might be some situations where we would mix sequencing types but probably not this one

Reply:
Yeah agreed, upgraded to an error